README.md

```diff
@@ -206,7 +235,14 @@ bordertech.config.resource.append=my-project.properties
 
 [ConfigurationLoader](https://github.com/BorderTech/java-config/blob/master/src/main/java/com/github/bordertech/config/ConfigurationLoader.java) is the [SPI](https://docs.oracle.com/javase/tutorial/sound/SPI-intro.html) interface for classes that can load a custom configuration.
 
-By default, the SPI lookup is enabled and if found, will be appended to the default implementation.
+By default, the SPI lookup is enabled and if found, it will create the custom configuration
+ 
+If the `bordertech.config.spi.append.default` is true the Default Configuration will also be appended to the configuration.
+
+### Best Practice
+
+THe Default Configuration allows you to set the profile `bordertech.config.profile` but you would normally set the profile through a
+JVM System Property or an OS environment variable.  
```

Comment:
Updated to:

When using java-config in a container and setting specific properties for that container instance, this can be achieved by property file(s) placed in the container that can be referred to and included within the application at runtime.